composer.json

```diff
@@ -44,8 +42,7 @@
     "extra": {
         "typo3/cms": {
             "extension-key": "cart",
-            "app-dir": ".build",
-            "web-dir": ".build/public"
+            "web-dir": ".build/web"
```

Comment:
Why switching to the kind of deprecated `/web` instead of using the newer `/public`?

Reply:
I am trying to standardize all my extensions bit by bit. In cart_products I use the subfolder web. Here a change to public would also be possible, but would mean a little more effort, because I would then have to adapt the paths for the acceptance tests.

Reply:
Got it! :)